### ISSUE TITLE
Qualtrics 

### ISSUE THREAD

**Issue by jeffhoye** [jeffhoye/evalulinks#3]
https://nyu.qualtrics.com/SE/?SID=SV_37ZTS1HfoXFaXpb

**Comment by NYUjason** [comment]
Let me know if there are any issues with the survey or questions

**Comment by NYUjason** [comment]
Hey Jeff: Here are some comments from My RAs who played with it. After you review it, perhaps it makes sense to chat next week, maybe Thursday or Frieday?

COMMENTS:

-          Could have used an alert to know when the other person’s content was ready. Was just sitting here and didn’t notice the change in text saying that I could move forward

In the discussion section:
          Are there group chats or all just dyads?
          Chat colors should be different for different users
          Maybe a dialogue box so the page doesn’t keep getting longer, concerned about different length conversations happening at the same time?
- The final assessment just kept coming up at the end, no way to tell if it was the end or not, the only way to end the session was to hit “quit”
- In one round when I got to the discussion section I got an error saying my name needs to be more than 3 characters (I had just put in my initials), after the error I just got a blank screen, no way to change it or move forward, perhaps this could have been an alert when I put my name in the box earlier.
